src/translations/maps/baseMap.js

```diff
@@ -0,0 +1,150 @@
+import { rangeEach } from '../../helpers/number';
+import { mixin } from '../../helpers/object';
+import { isFunction } from '../../helpers/function';
+import localHooks from '../../mixins/localHooks';
+
+/**
+ * Map for storing mappings from an index to a value.
+ */
+class BaseMap {
+  constructor(initValueOrFn = null) {
+    /**
+     * List of mappings.
```

Comment:
I suppose that instead of `List of mappings` it should say `List of values`?

Reply:
I have a new proposition.

Reply:
My proposition: https://github.com/handsontable/handsontable/pull/5945/commits/6e21a3a50ba2114035f8b236d90d7e523f096168#diff-bdb2b791e2e370abb7e7e9b05e12cbfbR11-R17